src/containerapp/azext_containerapp/tests/latest/test_containerapp_scenario.py

```diff
@@ -8,35 +8,176 @@
 import unittest
 
 from azure.cli.testsdk.scenario_tests import AllowLargeResponse
-from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
+from knack.util import CLIError
 
 
 TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
 
 
-@unittest.skip("Managed environment flaky")
+@live_only()
 class ContainerappScenarioTest(ScenarioTest):
     @AllowLargeResponse(8192)
     @ResourceGroupPreparer(location="centraluseuap")
```

Comment:
not all subs have permissions to EUAP regions, & these tests don't run on our subs so avoid using this region. Use our non-canary region for testing.

Reply:
Changed this to "eastus2"